### PR TITLE
Master v9

### PR DESCRIPTION
**Pull request by robertjf**
Lots of goodies here:

* Multi-targeting dotnet 5 & DotNet Framework 4.72
* Added Github Actions to Build and Deploy - Deploy needs a GitHub Secret added - `NUGET_API_KEY` containing a NuGet package api key with appropriate credentials
* Added a database to the sample UmbracoV9 project to store configuration - feel free to override - I've also added the username/password to the ReadMe.md file.
* Split Core code into a separate dll/NuGet package

Note: There's now a README.nuget.md file - this is used in the NuGet packages and excludes some of the stuff in the README.md file - feel free to update as required.

### CODE REVIEW

File: UmbracoV9/UmbracoV9.csproj [hunk 1]
```diff
@@ -46,5 +48,8 @@
         <RazorCompileOnBuild>false</RazorCompileOnBuild>
         <RazorCompileOnPublish>false</RazorCompileOnPublish>
     </PropertyGroup>
+	<Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+		<Exec Command="xcopy $(SolutionDir)Iconic\App_Plugins\Iconic\*.* $(ProjectDir)App_Plugins\Iconic\ /S /E /Y" />
+	</Target>
```
Comment (skartknet) [inline UmbracoV9/UmbracoV9.csproj:58]: The targets file that you removed a few lines above copies the package's files into the V9 project for testing already.

**Comment by robertjf** [comment]
@skartknet :
>Note: There's now a README.nuget.md file - this is used in the NuGet packages and excludes some of the stuff in the README.md file - feel free to update as required.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+name: Our.Umbraco.Icpnic - Release
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  SOLUTION: Our.Umbraco.Icpnic.sln
```
[the remark this answers is not in the document]
Comment (robertjf) [reply .github/workflows/release.yml:8]: I'd say there was a typo in a search/replace... will fix.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+name: Our.Umbraco.Icpnic - Release
```
[the remark this answers is not in the document]
Comment (robertjf) [reply .github/workflows/release.yml:1]: apart from the type in the Project Name, this is informational only and anything goes here!

File: README.nuget.md [hunk 4]
```diff
@@ -6,25 +6,12 @@ With Iconic you will be able to create a property editor that can use virtually
...
+Use V3 for Umbraco 8 & 9\
 Use V1 for Umbraco 7
 
-### Nuget
-[![NuGet](https://buildstats.info/nuget/Koben.Iconic)](https://www.nuget.org/packages/Koben.Iconic/)
-
-You can run the following command from within Visual Studio:
-
-    PM> Install-Package Koben.Iconic
-
```
[the remark this answers is not in the document]
Comment (robertjf) [reply README.nuget.md]: the README.nuget.md file is included in the NuGet package and used on the nuget.org listing - no need for this information here, it's still in the README.md file though.

File: UmbracoV9/UmbracoV9.csproj (hunk 1 above)
Comment (robertjf) [reply UmbracoV9/UmbracoV9.csproj:58]: true - both changes can be reverted...

File: .github/workflows/release.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (robertjf) [reply .github/workflows/release.yml]: Fixed now (new commits)

File: .github/workflows/release.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (robertjf) [reply .github/workflows/release.yml]: Wow this was horribly broken - fixed now - the solution and project paths/file names should now be in line with what's in the repo.

Side note: Some cleanup probably required for consistency of naming!  Are you moving to renaming the nuget packages to Our.Umbraco.Iconic, or keeping Koben.Iconic?  Projects/solution names should probably match whatever you choose...

File: README.nuget.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (skartknet) [reply README.nuget.md]: ahh, I see.

File: .github/workflows/release.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (skartknet) [reply .github/workflows/release.yml]: I would actually like to move it to Our.Umbraco.Iconic, but if I give it a different ID on Nuget I will lose the rest of the version hostory.

[the remark this answers is not in the document]
Comment (robertjf) [reply .github/workflows/release.yml]: yes, that's why I left it the way it is currently... I wonder if NuGet support renaming requests...

[the remark this answers is not in the document]
Comment (robertjf) [reply .github/workflows/release.yml]: you could also deprecate the old package and redirect it in the README perhaps...

**Pull request merged**